src/core/ast.h

```diff
@@ -1021,14 +1021,14 @@ class AST_LangPrimitive : public AST_expr {
 public:
     enum Opcodes {
         ISINSTANCE,
-        LANDINGPAD,
+        LANDINGPAD, // grabs the info about the last raised exception
         LOCALS,
         GET_ITER,
         IMPORT_FROM,
         IMPORT_NAME,
         IMPORT_STAR,
         NONE,
-        NONZERO,
+        NONZERO, // Python's __nonzero__ magic method
```

Comment:
This is close but not quite true :P  (it's more analogous to the `bool()` function)  if you're curious you can check out objmodel.cpp::nonzero() to see how the semantics are slightly different.